plugin-server/src/utils/posthog.ts

```diff
@@ -25,3 +29,38 @@ export const captureTeamEvent = (team: Team, event: string, properties: Record<s
         },
     })
 }
+
+// We use sentry-style hints rather than our flat property lisr all over the place,
+// so define a type for them that we can flatten internally
+export type Primitive = number | string | boolean | bigint | symbol | null | undefined
+export interface ExceptionHint {
+    level: SeverityLevel
+    tags: Record<string, Primitive>
+    extra: Record<string, any>
+}
+
+export function captureException(exception: any, hint?: Partial<ExceptionHint>, distinctId?: string): string {
+    //If the passed "exception" is a string, capture it as a message, otherwise, capture it as an exception
+    let sentryId: string
+    if (typeof exception === 'string') {
+        sentryId = captureSentryMessage(exception, hint)
+    } else {
+        sentryId = captureSentryException(exception, hint)
+    }
+
+    if (!distinctId) {
+        // If we weren't given a distinct_id, we randomly generate one
+        distinctId = new UUID7().toString()
+    }
```

Comment:
Good spot on this! We shouldn't need this. Fix in the next version: https://github.com/PostHog/posthog-js-lite/pull/390